Revit_Core_Engine/Query/LinkedElementsInView.cs

```diff
@@ -34,29 +33,46 @@ public static partial class Query
         /****              Public methods               ****/
         /***************************************************/
 
-        [Description("Return elements from link instance located in given view. Method returns all instance elements in the view scope (including hidden elements).")]
-        [Input("view", "ViewPlan to get the elements from.")]
-        [Input("linkInstance", "Revit Link Instance to get the elements from.")]
+        [PreviousVersion("7.3", "BH.Revit.Engine.Core.Query.LinkedElementsInView(Autodesk.Revit.DB.ViewPlan, Autodesk.Revit.DB.RevitLinkInstance, System.Collections.Generic.List<Autodesk.Revit.DB.ElementFilter>)")]
+        [Description("Return elements from the revit link instance located in the view scope.")]
```

Comment:
Does the method return the hidden elements too? If so, I would consider coming back to the description as in `develop`.

Also worth explaining that the view needs to belong to the host document

Reply:
Added 👍